go.mod

```diff
@@ -39,3 +36,5 @@ require (
 	golang.org/x/net v0.0.0-20211208012354-db4efeb81f4b
 	google.golang.org/grpc v1.42.0
 )
+
+replace github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.2.0
```

Comment:
Instead of using a replace directive, if we are going to depend on this fork we should update the import paths explicitly and require it like a normal dependency.

The replacement saves a little bit of work up-front, but makes ongoing maintenance harder. IMO we should pay the tax now rather than later.

Reply:
agree here, we can cut a new release of tm-db and then this won't be needed.

Reply:
It's doesn't even save work up front: replace directives aren't transitive, so this breaks your downstream consumers, unless they replicate the same thing in their `go.mod`.